src/code-editor/code-editor.component.ts

```diff
@@ -558,6 +589,7 @@ export class TdCodeEditorComponent implements OnInit, AfterViewInit, ControlValu
         value: this._value,
         language: this.language,
         theme: this._theme,
+        automaticLayout: this._automaticLayout,
```

Comment:
no, I tried to call the update method in monaco for it and it didn't take.  So you have to destroy the instance and then recreate it.  So I was going to add a function for that, but then thought, why would you want to be toggling this setting on and off anyway, so left it out and it is only a one time thing at creation, just like with what monaco is doing.